themes/my-theme/scripts/image.js

```diff
@@ -64,7 +64,7 @@ hexo.extend.tag.register('banner_ad', function (args) {
   var assetPath = args[0];
   var targetUrl = args[1];
 
-  return `<a class="course-image" href="${targetUrl}" rel="nofollow" aria-label="Dev Academy courses">
+  return `<a class="course-image" href="${targetUrl}" rel="nofollow" data-ph="banner-ad__link" aria-label="Dev Academy courses">
```

Comment:
`banner-ad` name is not a good idea -> might be removed by ad blockers!

```suggestion
  return `<a class="course-image" href="${targetUrl}" rel="nofollow" data-ph="banner-course__link" aria-label="Dev Academy courses">
```